src/deluge/gui/ui/sound_editor.cpp

```diff
@@ -900,6 +978,16 @@ ActionResult SoundEditor::padAction(int32_t x, int32_t y, int32_t on) {
 		return ActionResult::REMIND_ME_OUTSIDE_CARD_ROUTINE;
 	}
 
+	//used to convert column press to a shortcut to change Perform FX menu displayed
+	if (((getRootUI() == &performanceSessionView) || (getCurrentUI() == &performanceSessionView))
+	    && !Buttons::isShiftButtonPressed() && performanceSessionView.defaultEditingMode
```

Comment:
No I don't think so. These checks are only relevant when you're trying to call performanceSessionView::padAction from soundEditor. I only want to interact with the grid in performance view when you're in soundEditor if you're not holding shift and you're in defaultEditingMode and you're not editing parameter assignments. Otherwise I want to exit soundEditor completely if you press the grid in regular performance view or param editing mode (e.g. reach line 1053: exitCompletely();)